installer/requirements-3.6.txt

```diff
@@ -30,7 +29,7 @@ msoffice_decrypt
 olefile
 oletools
 openpyxl
-pandas==1.1.1
```

Comment:
It's because of a limitation in the businesstime library.

Reply:
https://github.com/ace-ecosystem/ACE/commit/26558f446d5e29c8b36081de1ecdf289cc5fcd25#diff-293c1a40b0cd891ec5700b8d68b7e5402cfa7b7fc875706e3ba43ae9f6eff053

I remember that it was an issue in that businesstime lib... something in pandas that the lib was expecting to be there. I don't remember what it was though. When this code arrives back at my doorstep I will look to fix it without restricting the pandas lib.